h2o-genmodel/src/main/java/hex/genmodel/algos/gam/GamRowToRawDataConverter.java

```diff
@@ -0,0 +1,32 @@
+package hex.genmodel.algos.gam;
+
+import hex.genmodel.GenModel;
+import hex.genmodel.easy.CategoricalEncoder;
+import hex.genmodel.easy.EasyPredictModelWrapper;
+import hex.genmodel.easy.RowData;
+import hex.genmodel.easy.RowToRawDataConverter;
+import hex.genmodel.easy.exception.PredictException;
+
+import java.util.Map;
+
+import static hex.genmodel.utils.ArrayUtils.nanArray;
+
+public class GamRowToRawDataConverter extends RowToRawDataConverter {
+  GamMojoModelBase _m;
+  public GamRowToRawDataConverter(GenModel m, Map<String, Integer> modelColumnNameToIndexMap, Map<Integer, CategoricalEncoder> domainMap, EasyPredictModelWrapper.ErrorConsumer errorConsumer, EasyPredictModelWrapper.Config config) {
+    super(m, modelColumnNameToIndexMap, domainMap, errorConsumer, config);
+    _m = (GamMojoModelBase) m;
+  }
+  
+  @Override
+  public double[] convert(RowData data, double[] rawData) throws PredictException {
+    if (data.containsKey(_m._gamColNamesCenter[0][0])) {
```

Comment:
This part should be dropped, as data in production do NOT contain those columns. The only meaningful way the "gamified" columns can reach this place is our tests. 

We can always remove the columns after GAM model training is finished by simply calling `trainBinomial.remove(gamCols);` - and test it on real-world-like data.